.ci/benchmarks.groovy

```diff
@@ -9,11 +9,16 @@ pipeline {
     AWS_ACCOUNT_SECRET = 'secret/observability-team/ci/elastic-observability-aws-account-auth'
     EC_KEY_SECRET = 'secret/observability-team/ci/elastic-cloud/observability-pro'
     BENCHMARK_ES_SECRET = 'secret/apm-team/ci/benchmark-cloud'
+    KIBANA_SECRET = 'secret/k8s/elastic-apps/oauth-proxy/prod/domains/observability-benchmarks.elastic.dev'
+    PNG_REPORT_FILE = 'out.png'
+    KIBANA_DASHBOARD_URL = 'https://observability-benchmarks.elastic.dev/app/r/s/afdFh'
```

Comment:
I'd not expose the Kibana URL but use the Vault secret in favour of more dynamism